lib/restaurant.rb

```diff
@@ -0,0 +1,20 @@
+
+class Restaurant
+  def initialize ( messenger, menu)
+    @messenger = messenger
+    @menu = menu
+    @header = { :from => '+441290211165', :to => '+447527994300'}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)